main.go

```diff
@@ -29,60 +22,81 @@ func dbConn() (db *sql.DB) {
 }
 
 func Index(w http.ResponseWriter, r *http.Request) {
+
+	n := Funcionario{}
+	SlcFuncionario := []Funcionario{}
+	IndxPg := IndexPage{}
+
 	db := dbConn()
-	selDB, err := db.Query("SELECT *  FROM funcionarios ")
+	resultado, err := db.Query("SELECT *  FROM funcionarios ")
 	if err != nil {
 		panic(err.Error())
 	}
 
-	n := Names{}
-	res := []Names{}
-
-	for selDB.Next() {
+	for resultado.Next() {
 		var id int
 		var name, email string
+		var salario float64
 
-		err = selDB.Scan(&id, &name, &email)
+		err = resultado.Scan(&id, &name, &email, &salario)
 		if err != nil {
 			panic(err.Error())
 		}
 
-		n.Id = id
-		n.Name = name
-		n.Email = email
+		n.id = id
+		n.name = name
+		n.email = email
+		n.salario = salario
 
-		res = append(res, n)
-	}
+		SlcFuncionario = append(SlcFuncionario, n)
 
-	tmpl.ExecuteTemplate(w, "Index", res)
+	}
 
+	count := len(SlcFuncionario)
+	IndxPg.funcionarios = SlcFuncionario
+	IndxPg.count = count
+	tmpl.ExecuteTemplate(w, "Index", IndxPg)
 	defer db.Close()
 }
 
+type Funcionario struct {
```

Comment:
also, english for Funcionario, also, change the table name to the same

Reply:
ok